data/json/recipes/recipe_food.json

```diff
@@ -6639,6 +6661,29 @@
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
     "components": [ [ [ "ketchup", 2 ] ], [ [ "seasoning_mild", 2, "LIST" ] ], [ [ "sugar_standard", 1, "LIST" ] ], [ [ "vinegar", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "honey_mustard",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "17 m",
+    "charges": 3,
```

Comment:
Based on the quantity of materials you probably want more like 5 or 6 charges here.